Frontend/ui-driver/src/Flow.purs

```diff
@@ -252,6 +254,43 @@ onBoardingFlow = do
       else if (resp.rcVerificationStatus == "NO_DOC_AVAILABLE") then addVehicleDetailsflow
         else applicationSubmittedFlow "StatusScreen"
 
+updateDriverVersion :: Maybe Version -> Maybe Version -> FlowBT String Unit
```

Comment:
do the same for driver as well

Reply:
@Akash2172 also, We can move this function to common as this is unnecessary redundant